src/main/java/org/opendatakit/suitcase/ui/SuitcaseCLI.java

```diff
@@ -326,8 +324,12 @@ private Operation parseArgs(String[] args, Options options) {
       
       updateLogPath = line.getOptionValue(UPDATE_LOG_PATH_OPT);
 
-      version = line.getOptionValue(DATA_VERSION_OPT);
-
+      if (!line.hasOption(DATA_VERSION_OPT)) {
+        version=default_data_version;
+      }
+      else {
+        version = line.getOptionValue(DATA_VERSION_OPT);
```

Comment:
You could supply the default value directly to `getOptionValue` see an example on https://github.com/odk-x/suitcase/pull/49/files#diff-9851d999d0c7a8e7293d5cbb8cf98908b843b59ce1485c203537e99dbbcb9684L325